main.py

```diff
@@ -0,0 +1,63 @@
+import os
+from skimage.io import imread
+from skimage.transform import resize
+import numpy
+from sklearn.decomposition import PCA
+from sklearn.model_selection import train_test_split
+from sklearn.svm import SVC
+from sklearn.model_selection import GridSearchCV
+from sklearn.multiclass import OneVsRestClassifier
+from sklearn.metrics import accuracy_score
+
+training_path = os.path.join(__file__, '..\sample-images\\training-images')
```

Comment:
Let's modify this to allow users to provide their own path for training and test images, it doesn't need to be hardcoded